include/luxrays/utils/utils.h

```diff
@@ -41,6 +41,10 @@
 #define isnanf(a) _isnan(a)
 #endif
 
+#if (defined(__GNUC__) && (__GNUC__ > 5))
+using namespace std;
```

Comment:
My personal rule is to never have a "using namspace" in an header file. Please, remove this from the patch.